resolwe/flow/managers/dispatcher.py

```diff
@@ -267,17 +267,14 @@ def _include_environment_variables(self, program):
         export_commands = ['export {}="{}"'.format(key, value.replace('"', '\"')) for key, value in env_vars.items()]
         return os.linesep.join(export_commands) + os.linesep + program
 
-    def run(self, data, dest_dir, argv, run_sync=False, verbosity=1):
+    def run(self, data, dest_dir, argv, verbosity=1):
         """Select a concrete runner and run the process through it.
 
         :param data: The :class:`~resolwe.flow.models.Data` object that
             is to be run.
         :param dest_dir: The directory the
             :class:`~resolwe.flow.models.Data` object should be run from.
         :param argv: The argument vector used to spawn the executor.
-        :param run_sync: If ``True``, the method will not return until
```

Comment:
Can you extend the comment message with an explanation about why we want and can get rid of `run_sync` flag?

Reply:
The commit log already says the flag is disused. By definition, this means there's no code anywhere that uses it, so it's redundant and can be removed, which the commit does.

Reply:
I guess I read too quickly and missed the *disused* part 🙂 .